pages/apim/3.x/installation-guide/configuration/reporters/installation-guide-reporters.adoc

```diff
@@ -0,0 +1,201 @@
+= Reporters
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_installguide_reporters.html
+:page-folder: apim/installation-guide/reporters
+:page-description: Gravitee.io API Management - Reporters
+:page-keywords: Gravitee.io, API Platform, API Management, API Gateway, reporters, events, elasticsearch, json, message_pack, tcp
+:page-layout: apim3x
+
+
+
+Reporters are designed to write a variety of events occuring in the gatewy to a variety of outputs and format, giving you the ability to manage this data using a solution of your choice.
```

Comment:
```suggestion
Reporters are designed to write a variety of events occuring in the gateway to a variety of outputs and format, giving you the ability to manage this data using a solution of your choice.
```

Reply:
Good catch @gcusnieux - there was a newer edit in my side but not sure why it was not incorporated, see https://github.com/gravitee-io/gravitee-docs/pull/816#discussion_r907785606 note the additional rewording. We should use this version.